docs/index.rst

```diff
@@ -25,7 +25,8 @@ The goal is a common framework for physics experiments, so:
 - physics experiments can take advantage of modern software and best practices
 
 Read `15 minutes to Qcodes <https://qcodes.github.io/Qcodes/examples/15_minutes_to_QCoDeS.html>`__
-to have quick overview on QCoDeS.Many more elaborate examples can be found in the example notebooks.
+to have quick overview on QCoDeS. Many more elaborate examples can be found in the
+`example notebooks <http://qcodes.github.io/Qcodes/examples/index.html>`__ .
```

Comment:
In rst documents (except the readme which is also available outside the docs) you should use rst links and not links to the webpage. This link will not take you to your local notebook in case you are building locally. 

See here https://nbsphinx.readthedocs.io/en/0.4.2/a-normal-rst-file.html for some info

Reply:
Thanks Jens! I have referenced them to the local notebook.